pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -105,5 +105,20 @@ func status() registry.State {
 		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
 	}
 
+	// Ensure user is either a Windows Administrator or a Hyper-V Administrator.
+	adminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")`)
+	adminCheckOut, _ := adminCheckCmd.CombinedOutput()
+
+	hypervAdminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(([System.Security.Principal.SecurityIdentifier]::new("S-1-5-32-578")))`)
+	hypervAdminCheckOut, _ := hypervAdminCheckCmd.CombinedOutput()
+
+
+	if (strings.TrimSpace(string(adminCheckOut)) != "True") && (strings.TrimSpace(string(hypervAdminCheckOut)) != "True") {
+		err := fmt.Errorf("Hyper-v commands have to be run as an Administrator")
```

Comment:
Suggestion: `"Hyper-V requires Administrator privileges"`

The user may not know that commands are being invoked.